src/styles/includes/_getting-started-section-header.scss

```diff
@@ -0,0 +1,37 @@
+.section-title {
+  font-family: 'Roboto';
+  font-style: normal;
```

Comment:
can we apply this directly to `.section-header-wrapper` instead?

Reply:
good call, Ill nest the styling as well